stubs/iterable.stub

```diff
@@ -436,3 +436,23 @@ class RegexIterator extends FilterIterator {
      */
     public function key() {}
 }
+
+/**
+ * @template-implements Iterator<void, void>
```

Comment:
Is it ok to use the type `void` in this case?

Reply:
I think `never` is more fitting. And if you find yourself testing something with `dumpType`, please add a new testcase for NodeScopeResolverTest so that we test it continuously (with assertType calls).

Reply:
Fixed!